src/utils/DashboardHelperFunctions.js

```diff
@@ -1,40 +1,57 @@
 import axios from 'axios';
 
 export const putIncidents = (oktaAxios, incidents, status) => {
-  const reviewedIncidents = incidents.map(incident => {
-    return {
-      status: status,
-      city: incident.city,
-      state: incident.state,
-      force_rank: incident.force_rank,
-      lat: incident.lat,
-      long: incident.long,
-      incident_id: incident.incident_id,
-    };
+  // const reviewedIncidents = incidents.map(incident => {
+  //   return {
+  //     status: status,
+  //     city: incident.city,
+  //     state: incident.state,
+  //     force_rank: incident.force_rank,
+  //     lat: incident.lat,
+  //     long: incident.long,
+  //     incident_id: incident.incident_id,
+  //   };
+  // });
```

Comment:
Could you remove this commented code? Thank you for using the spread operator by the way instead of making me read "incident." over and over